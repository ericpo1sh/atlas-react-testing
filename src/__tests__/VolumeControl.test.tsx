import { render } from "@testing-library/react";
import { expect, test } from "vitest";
import VolumeControls from "../components/VolumeControls";

test("Volume Controls Renders Correctly", () => {
  const { container } = render(<VolumeControls/>);
  expect(container).not.toBeNull();
  expect(container).toMatchSnapshot();
});
// I dont have props in my VolumeControls Component, so I cant test for them.
