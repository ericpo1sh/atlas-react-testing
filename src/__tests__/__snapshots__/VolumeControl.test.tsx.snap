// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Volume Controls Renders Correctly 1`] = `
<div>
  <div
    class="flex-col flex items-center mt-6 pb-1.5"
  >
    <div
      class="w-96 flex justify-between gap-4"
    >
      <img
        alt="Volume icon"
        class="w-5"
        src="/src/assets/volume.svg"
      />
      <input
        class="w-full cursor-pointer caret-jacarta-800 bg-jacarta-800 accent-jacarta-800 rounded-lg"
        max="100"
        min="0"
        type="range"
      />
    </div>
  </div>
</div>
`;
